ssm-automation/createRoles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Replace with your AWS account ID
+AWS_ACCOUNT="1234567890"
+
+# Replace with the number of approver roles you want to create
+NUMBER_OF_REQUEST_APPROVERS=2
+
+echo "Creating CreatePAARole..."
+aws iam create-role --role-name CreatePAARole --assume-role-policy-document file://policies/SSMTrustPolicy.json
```

Comment:
Don't see policies/SSMTrustPolicy.json file, missing?

Reply:
Hmm somehow the policies didnt make it into the commit - I will add those